test/component_failures_test.py

```diff
@@ -23,6 +24,65 @@ def ray_start_workers_separate():
     ray.shutdown()
 
 
+@pytest.fixture
+def shutdown_only():
+    yield None
+    # The code after the yield will run as teardown code.
+    ray.shutdown()
+
+
```

Comment:
The two tests here are copied for xray since they are slightly different between legacy ray and xray (one difference is that the xray version does not use custom object IDs).